docs/docs/migration-guide.mdx

```diff
@@ -13,6 +13,66 @@ This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
 
+## Rasa 3.2 to 3.3
+
+### ConcurrentRedisLockStore
```

Comment:
Not entirely sure how this section ended up here, maybe you merged an old version of `main` into your branch, however this needs to be removed (Lines 18 - 43 inclusive).